src/components/cart/OrderSummary/order-summary.scss

```diff
@@ -0,0 +1,24 @@
+@import "../../../styles/scaffold";
+
+.order-summary[data-order-summary] {
+  padding: 1rem;
+
```

Comment:
Maybe we can simplify like this:

```
[data-order-summary-subtotal], 
[data-order-summary-discount],
[data-order-summary-total] {
     display: flex;
     justify-content: space-between;
     line-height: 1.5;
   }

   [data-order-summary-discount] {
     color: green;
   }

```
 or we could have data-attribute only for `[data-order-summary-discount]`
 
```
p {
     display: flex;
     justify-content: space-between;
     line-height: 1.5;
   }
 
    [data-order-summary-discount] {
     color: green;
   }
```


What do you think?